grafanimate/grafana.py

```diff
@@ -38,12 +42,17 @@ def boot_grafana(self):
         Navigate to Grafana application and inject Grafana Sidecar service.
         """
         log.info("Starting Grafana at %s", self.baseurl)
-        self.set_window_size(1920, 1080)
-
+        if self.window_size:
+            self.set_window_size(
+                int(self.window_size[0]),
+                int(self.window_size[1] + (85 * self.zoom_factor)),
+            )
```

Comment:
Is that ` height + 85 * self.zoom_factor` formular correct? Can you elaborate, why "85"?